src/sage/groups/abelian_gps/abelian_group.py

```diff
@@ -976,13 +978,15 @@ def gens_orders(self):
             sage: Z2xZ3 is Z6
             False
 
-        TESTS::
 
+        TESTS::
+            sage: G, (g0, g1) = AbelianGroup(2, [48, 0]).objgens()
```

Comment:
```suggestion
        TESTS::

            sage: G, (g0, g1) = AbelianGroup(2, [48, 0]).objgens()
```
This needs a blank line between as it was before the change.